docs/working-with-gatekeeper.md

```diff
@@ -0,0 +1,473 @@
+   ```
+
+   Deploy namespace: https://github.com/open-policy-agent/gatekeeper/blob/0d239574f8e71908325391d49cb8dd8e4ed6f6fa/deploy/gatekeeper.yaml#L1-L9
+   
+   Deploy Gatekeeper CRDs: https://github.com/open-policy-agent/gatekeeper/blob/0d239574f8e71908325391d49cb8dd8e4ed6f6fa/deploy/gatekeeper.yaml#L27-L1999
+   
+   Deploy Gatekeeper secrets: https://github.com/open-policy-agent/gatekeeper/blob/0d239574f8e71908325391d49cb8dd8e4ed6f6fa/deploy/gatekeeper.yaml#L2261-L2267
+   
+   Deploy webhook config:
+   ```yaml
+   apiVersion: admissionregistration.k8s.io/v1
+   kind: MutatingWebhookConfiguration
+   metadata:
+     labels:
+       gatekeeper.sh/system: "yes"
+     name: gatekeeper-mutating-webhook-configuration
+   webhooks:
+     - admissionReviewVersions:
+         - v1
+         - v1beta1
+       clientConfig:
+         url: https://gatekeeper-webhook-service.gatekeeper-system.svc:443/v1/mutate
+       failurePolicy: Ignore
+       matchPolicy: Exact
+       name: mutation.gatekeeper.sh
+       namespaceSelector:
+         matchExpressions:
+           - key: admission.gatekeeper.sh/ignore
+             operator: DoesNotExist
+       rules:
+         - apiGroups:
+             - '*'
+           apiVersions:
+             - '*'
+           operations:
+             - CREATE
+             - UPDATE
+           resources:
+             - '*'
+       sideEffects: None
+       timeoutSeconds: 1
+   ---
+   apiVersion: admissionregistration.k8s.io/v1
+   kind: ValidatingWebhookConfiguration
+   metadata:
+     labels:
+       gatekeeper.sh/system: "yes"
+     name: gatekeeper-validating-webhook-configuration
+   webhooks:
+     - admissionReviewVersions:
+         - v1
+         - v1beta1
+       clientConfig:
+         url: https://gatekeeper-webhook-service.gatekeeper-system.svc:443/v1/admit
+       failurePolicy: Ignore
+       matchPolicy: Exact
+       name: validation.gatekeeper.sh
+       namespaceSelector:
+         matchExpressions:
+           - key: admission.gatekeeper.sh/ignore
+             operator: DoesNotExist
+       rules:
+         - apiGroups:
+             - '*'
+           apiVersions:
+             - '*'
+           operations:
+             - CREATE
+             - UPDATE
+           resources:
+             - '*'
+       sideEffects: None
+       timeoutSeconds: 3
+     - admissionReviewVersions:
+         - v1
+         - v1beta1
+       clientConfig:
+         url: https://gatekeeper-webhook-service.gatekeeper-system.svc:443/v1/admitlabel
+       failurePolicy: Fail
+       matchPolicy: Exact
+       name: check-ignore-label.gatekeeper.sh
+       rules:
+         - apiGroups:
+             - ""
+           apiVersions:
+             - '*'
+           operations:
+             - CREATE
+             - UPDATE
+           resources:
+             - namespaces
+       sideEffects: None
+       timeoutSeconds: 3
+   ```
+   You need to change the clientconfig from service type to url type for multi-cluster deployment.
```

Comment:
Can we use `git diff`(or json path, .webhooks[*].clientConfig.url) to highlight the necessary changes for the webhook?

Reply:
Add notes for it.